v2/pkg/core/workflow_execute.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/projectdiscovery/nuclei/v2/pkg/workflows"
 )
 
+const workflowExecutionErrorMessageTemplate = "[%s] Could not execute workflow step: %s\n"
```

Comment:
I'd opt for something shorter, for example `workflowExecutionFailure`

Reply:
> I'd opt for something shorter, for example workflowExecutionFailure

But that still doesn't say that it expects a parameter, hence it can lead to bugs. I'd rather have longer and self-explanatory constant and variable names than to have to jump back and forth to understand what value it should hold, or to make arbitrary formatting mistakes. If you really want (although I don't necessarily recommend it), we could use abbreviations for certain words like "Message" -> "Msg", but in this case we'd need to be careful not to shorten words that might have multiple meanings e.g. "Temp" should not refer to "Template", because people would rather expect it to mean "Temporary".

From the go [clean code](https://github.com/golang/go/wiki/CodeReviewComments#variable-names): _"the further from its declaration that a name is used, the more descriptive the name must be"_

The code is written to be read by humans :)

Reply:
The go standard library has a lot of abbreviations such as `err`, `msg`. Most IDEs should support constants preview, so I think it's visible that the variable contains formatting directives:
<img width="957" alt="Screenshot 2022-05-10 at 10 30 23" src="https://user-images.githubusercontent.com/13421144/167585489-0463bfc1-de79-470f-8c05-a01c2937039b.png">

Reply:
I'd opt for using a shorter form (like the suggested `workflowExecutionFailure`) if the name gets very long (like in this case) or contains redundant negations verb/terms.